packages/CollabCodeEdit-Core.package/CCEMessage.class/class/sendEdit..st

```diff
@@ -1,4 +1,6 @@
 client-Message creation
 sendEdit: aCCEEdit
-	^ self new: (CCEMessage clientMsgTypes at:#sendMulticastToSession:from: ) 
-			with: (STON toString: aCCEEdit)
\ No newline at end of file
+	^ self new 
+			type: (CCEMessage clientMsgTypes at: #sendEdit:from:); 
```

Comment:
is there a reason you separate the new call from the setter methods?

Reply:
That is how cascades should look like.